src/groups/group_action.jl

```diff
@@ -112,6 +123,21 @@ function apply_diff!(A::AbstractGroupAction, Y, a, p, X)
     )
 end
 
+@doc raw"""
+    apply_diff_group(A::AbstractGroupAction, a, X, p)
+
+For a point on manifold ``p ∈ \mathcal M`` and an element `X` of the tangent space at `a`,
```

Comment:
```suggestion
For a point on manifold ``p ∈ \mathcal M`` and an element `X` of the tangent space at `a ∈ A`
```

Reply:
Do we indicate Lie groups by letters like `A`?

Reply:
No, usually `G`, `M` or `H` if there is more than one. I'll rephrase it.

Reply:
Cool, yeah I also had G in mind, and making sure to know that p is from M and a is from G would be nice as well (or is G=M here?).

Reply:
No, `M` is different as usually when it comes to actions.

Reply:
Ok, that was also something that did not get that clear (since also `M` is not part of the parameters).

Reply:
So do you think the new text is clear enough?

Reply:
Nearly. It is still not clear where `M` comes from (it is merely implicitly given by using some `p`?) , since it is not a parameter of the function (and probably should not become that?), so maybe clarifying that a little would even improve the docs, the rest is fine now.

Reply:
I've rewritten it again, hopefully it's better now :slightly_smiling_face: .

Reply:
nearly (just something small at another line) – and sorry for being a little picky/precise here.